### ISSUE TITLE
Add a mechanism for doing long quiescent pcap

### ISSUE THREAD

**Issue by pisuke** [faucetsdn/daq#97]

**Comment by grafnu** [comment]
Ok, the [monitor_scan_sec](https://github.com/faucetsdn/daq/blob/master/misc/system.conf#L65) config option is already there. It defaults to 30s, but can be set to whatever necessary. Then it can be accessed through a file, [something like but not exactly the same as this],(https://github.com/faucetsdn/daq/blob/master/misc/runtime_configs/discover/port-01/ping_runtime.sh#L2) in a docker test.

I'm guessing there'll be some interaction with DHCP that needs to be worked out, since with a 2m lease time it'll surely be communicating quite rapidly, but that would change with a 1h lease time. So, I think there will be a dependency on this test (quiescent activity) on the [production DHCP capability issue](https://github.com/faucetsdn/daq/issues/105).

This should be enough to develop the test, and make sure that everything necessary is there -- aka "unblocking" and requiring feedback from your end if this is sufficient (because I wager it is). The next steps for this would be to develop the test that "always passes" b/c of DHCP.

**Comment by pisuke** [comment]
Thanks Trevor, giving it a try with Ed shortly.

**Comment by pisuke** [comment]
Initial tests implementation in [PR 119](https://github.com/grafnu/daq/pull/119)

**Comment by grafnu** [comment]
I'm closing this issue out unless there's something specifically I should be doing for this.  If the tests are captured in PR 119 then things can be addressed there.  If there's something new that I need to do then either re-open this or file another issue.
